gnomad_qc/v5/sample_qc/merge_project_meta.py

```diff
@@ -137,10 +157,10 @@ def get_config() -> Dict[str, Dict[str, hl.expr.Expression]]:
                         "project_mean_depth": "mean_coverage",
                         "project_bases_over_20x_coverage": "genome_coverage",
                         # TODO: Do we want dragen estimates or verifybamid? They show
-                        # the two metrics correlate in the QC report
+                        # the two metrics correlate in the QC report.all
```

Comment:
```suggestion
                        # the two metrics correlate in the QC report
```
was this comment to be extended?